openshift/e2e-tests-openshift.sh

```diff
@@ -16,6 +16,7 @@ readonly TEST_NAMESPACE=serving-tests
 readonly TEST_NAMESPACE_ALT=serving-tests-alt
 readonly SERVING_NAMESPACE=knative-serving
 readonly TARGET_IMAGE_PREFIX="$INTERNAL_REGISTRY/$SERVING_NAMESPACE/knative-serving-"
+readonly OLM_NAMESPACE="openshift-operator-lifecycle-manager"
```

Comment:
They change this name so often that I might consider something along the lines of...
`OLM_NAMESPACE=$(kubectl get pods --all-namespaces | grep olm-operator | head -1 | awk '{print $1}')`

Reply:
I'd honestly rather punt on this, given Openshift 4 is released now it shouldn't change as often. I shall stand corrected.